### PR TITLE
Feature/teleporters

### PR DESCRIPTION
**Pull request by Parker-CP**
Added Portals that teleport the player to a specific location on the map.
- Added Portals to the map
- Portals teleport the player to a predefined location
- Removed artifacts of old port

### CODE REVIEW

File: lib/player.js [hunk 1]
```diff
@@ -22,6 +22,9 @@ class Player {
     } else if(targetCell.value === 2) {
       this.position.setValue(1);
       this.setPosition(targetCell);
+    } else if(targetCell.value === 3) {
+      this.position.setValue(1);
+      this.setPosition(targetCell.locationCell)
```
Comment (rgbatty) [inline lib/player.js]: Could we use `targetCell.portalTarget` as the name instead?

File: lib/generator.js [hunk 2]
```diff
@@ -85,6 +86,18 @@ class Generator {
...
 
+    generatePortals(n) {
+      for( var i = 0; i < n; i ++) {
+        var room = this.grid.rooms[Math.floor(Math.random() * this.grid.rooms.length)];
+        var cell = room.cells[Math.floor(Math.random() * room.cells.length)];
+        var secondroom = this.grid.rooms[Math.floor(Math.random() * this.grid.rooms.length)];
+        var targetCell = secondroom.cells[Math.floor(Math.random() * secondroom.cells.length)];
+
+        cell.setValue(3);
+        cell.locationCell = targetCell
```
Comment (rgbatty) [inline lib/generator.js]: Related to my other comment, can we swap this to `cell.portalTarget`?

**Comment by Parker-CP** [comment]
Yep we sure can!

File: lib/generator.js [hunk 3]
```diff
@@ -85,6 +86,18 @@ class Generator {
       return cell;
     }
 
+    generatePortals(n) {
```
Comment (rgbatty) [inline lib/generator.js]: Can we use something other than `n` for a variable name? its not very descriptive.

**Comment by rgbatty** [comment]
Thanks for that new commit. This looks great. Good work. 👍

**Pull request merged**